example/mnist/dataset.yaml.link

```diff
@@ -0,0 +1,15 @@
+name: mnist
```

Comment:
dataset build cli supports build without dataset.yaml.

```
swcli dataset build . --handler mnist.dataset:DatasetProcessExecutor --name mnist-link
```
so we can drop the dataset.yaml.link and dataset.yaml.raw files.

Reply:
done